nss-day-cohort-46/reactive-nutshell-garment-of-illusions-1
# added bootstrap styling to friends and events

# Description

changed the styling on friends and events, but functionality should be the same

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

look at the friends and events sections, and see if the bootstrap styling shows up and functionality remains.

you have to click on the item to see the buttons that were there before.

there is an error that shows up when you expand a column, but it is because we are running the app in "strict mode" in the index.js file. the error goes away if you remove that part, but that's pretty scary.

`git fetch --all`
`git checkout pp-friends-bootstrap`
`npm start`

In another tab, cd into `api` and run

`json-server -p 8088 -w database.json`

# Checklist:
